boa3/internal/model/builtin/native/nativecontract.py

```diff
@@ -17,43 +17,80 @@ class NativeContract:
     RoleManagement = RoleManagementClass.build()
     StdLib = StdLibClass.build()
 
+    # Deprecated Native Contract Names
+    gas_name = 'GAS'
+    ledger_name = 'Ledger'
+    neo_name = 'NEO'
+    oracle_name = 'Oracle'
+    policy_name = 'Policy'
+
+    GAS_ = GAS.clone()
+    GAS_._identifier = gas_name
+    Ledger_ = Ledger.clone()
+    Ledger_._identifier = ledger_name
+    NEO_ = NEO.clone()
+    NEO_._identifier = neo_name
+    Oracle_ = Oracle.clone()
+    Oracle_._identifier = oracle_name
+    Policy_ = Policy.clone()
+    Policy_._identifier = policy_name
+
```

Comment:
for the classes that were renamed you can set the id on the package definition instead of here (see next comments)